src/pages/Homepage/LendByCategoryHomepage.vue

```diff
@@ -323,6 +328,51 @@ export default {
 			},
 		};
 	},
+	methods: {
+		assemblePageViewData() {
+			// eslint-disable-next-line max-len
+			const schema = 'https://raw.githubusercontent.com/kiva/snowplow/master/conf/snowplow_category_row_page_load_event_schema_1_0_4.json#';
+			const loanIds = [];
+			const pageViewTrackData = { schema, data: {} };
+
+			// Gathering the category id and loan id from the Featured Loans section to pass through for tracking
+			const featuredCategoryId = _get(this, '$refs.featuredLoansCarousel.prefetchedCategoryInfo[0].id');
+			const featuredCategoryLoan = this.$refs.featuredLoansCarousel.featuredLoanForCategory(featuredCategoryId);
+
+			// The schema we're using requires a categorySetIdentifer to be set,
+			// hardcoding to default because there is no experiment A/B for this
+			pageViewTrackData.data.categorySetIdentifier = 'default';
+
+			// Formatting and pushing featured loans info into loanIds
+			loanIds.push({
+				r: 0, p: 1, c: featuredCategoryId, l: featuredCategoryLoan.id
+			});
+
+			// Gathering the category ids and loan ids from the Loan Category section to pass through for tracking
+			const loanCategoryId = _get(this, '$refs.loanCategoriesSection.prefetchedCategoryInfo[0].id');
+			const categoryLoans = this.$refs.loanCategoriesSection.getCategoryLoans(loanCategoryId);
+
+			// Formatting and pushing Loan Category info into loanIds
+			_each(categoryLoans, (loan, loanIndex) => {
```

Comment:
We can replace _each here with a native .forEach `categoryLoans` is just an array.